spring-ws-security/src/test/java/org/springframework/ws/soap/security/wss4j2/callback/SpringSecurityPasswordValidationCallbackHandlerTest.java

```diff
@@ -62,6 +73,15 @@ public void setUp() {
 
 		passwordCallback = new WSPasswordCallback("Ernie", null, "type", WSPasswordCallback.USERNAME_TOKEN);
 	}
+	
+	/**
+	 * add a tearDown method at the end, 
+	 * to clean {@code SecurityContextHolder} after this class
+	 */
+	@AfterAll
+	public static void afterClassTearDown() { 
+		SecurityContextHolder.clearContext();
+	}
```

Comment:
I don't think we need this if we are clearing the context before each test method.

Reply:
Exactly. Deleted this method per your suggestion. Thanks!